src/player.cpp

```diff
@@ -3751,6 +3751,18 @@ void Player::addAttacked(const Player* attacked)
 	attackedSet.insert(attacked->guid);
 }
 
+void Player::removeAttacked(const Player* attacked)
+{
+	if (!attacked || attacked == this) {
+		return;
+	}
+
+	auto it = attackedSet.find(attacked->guid);
```

Comment:
I think it's better to be safe than sorry, as later it can be used in other context if need be